etc/fle2_aead_selftest.py

```diff
@@ -0,0 +1,40 @@
+# payloads_test.py is a self-test of payloads.py.
```

Comment:
Should this comment be updated? There is no `payloads_test.py` or `payloads.py` file.